src/careamics/models/lvae/noise_models.py

```diff
@@ -410,3 +473,86 @@ def getSignalObservationPairs(self, signal, observation, lowerClip, upperClip):
     def forward(self, x, y):
         """Temporary dummy forward method."""
         return x, y
+
+    # TODO taken from pn2v. Ashesh needs to clarify this
```

Comment:
It looks alright. I basically compared it with https://github.com/ashesh-0/PPN2V/blob/master/src/ppn2v/pn2v/gaussianMixtureNoiseModel.py#L257 and it looks quite similar. If you did not change anything (other than jointloss removal) then it is all good.